api/app/views/mno_enterprise/jpi/v1/app_instances/_resource.json.jbuilder

```diff
@@ -23,6 +23,7 @@ end
 #   json.connector_version app_instance.oauth_keys[:version]
 # end
 
+app_instance.without_tenant = true if params[:data] && params[:data] == 'unscoped'
```

Comment:
You could use `unscoped` as the params and it'd be:
`app_instance.without_tenant = params[:unscoped].present?`